UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/FunctionalTests/ExchangeSetByProductIdentifiers.cs

```diff
@@ -32,7 +31,7 @@ public async Task SetupAsync()
         [Test]
         public async Task WhenICallTheApiWithOutAuthToken_ThenAnUnauthorisedResponseIsReturned()
         {
-            ProductIdentifiermodel.ProductIdentifier = new List<string>() { "GB123456", "GB160060", "AU334550" };
+            ProductIdentifiermodel.ProductIdentifier = new List<string>() { "PE62263A", "FR373890", "CN584233" };
```

Comment:
If we're going to be using same cells can we extract this out into a field

Reply:
Also Im not sure if this is correct as I thought the only data that was verified in QA was the German cells that start with "DE"

Reply:
Hi Rob, I have done the changes for other endpoints as well with German cells. Thanks